docs/templates/systemd-in-docker.md

```diff
@@ -0,0 +1,115 @@
+The [Sysbox](https://github.com/nestybox/sysbox) container runtime allows unprivileged users to run system-level applications, such as Systemd, securely from the workspace containers. Sysbox requires a [compatible Linux distribution](https://github.com/nestybox/sysbox/blob/master/docs/distro-compat.md) to implement these security features.
```

Comment:
It's unfortunate how much of this doc is duplicated with the `docker-in-docker.md`. Perhaps we can make the meat of it a suffix to the `docker-in-docker.md` doc?